server/api/batch_inventory.py

```diff
@@ -68,13 +93,20 @@ def items_list_to_dict(items):
 def process_batch_inventory_cvr_file(
     jurisdiction_id: str, user: Tuple[UserType, str], support_user_email: Optional[str],
 ):
-    jurisdiction = Jurisdiction.query.get(jurisdiction_id)
-    batch_inventory_data = BatchInventoryData.query.get(jurisdiction_id)
+    working_directory = tempfile.mkdtemp()
 
-    def process():
-        contests = list(jurisdiction.contests)
+    def clean_up_file_system():
+        if os.path.exists(working_directory):
+            shutil.rmtree(working_directory)
```

Comment:
This happens to be safe for our system (and probably for any *nix), but is vulnerable to symlink attacks on non-fd based file systems ([docs](https://docs.python.org/3.12/library/shutil.html#shutil.rmtree.avoids_symlink_attacks)). From our staging console:

```
>>> shutil.rmtree.avoids_symlink_attacks
True
```

Reply:
Thank you for checking that! I'm still having some trouble envisioning how someone would exploit this specific call if we weren't on an fd-based file system. But maybe that's not quite what you're saying - just pointing out the risk that comes with `shutil.rmtree` in general